.gitignore

```diff
@@ -30,3 +30,6 @@
 *.exe
 *.out
 *.app
+
+# Build folders
+build/
```

Comment:
When you open the project in clion, the default build folder is cmake-build-debug. We should probably gitignore that folder as well. Not sure what the visual studio default is - will check.

Reply:
We can either gitignore that or you can go into the settings in clion and change the name of the generated build folder, but either way it doesn't hurt to add to the gitignore so will do